gsoc.yml

```diff
@@ -1,3 +1,34 @@
+gsoc2020:
+  theand9:
+    rss_fed: "rss-url" # TODO (add blog with gsoc tag)
+    project: 'stingray'
+  pkj-m:
+    rss_feed: "https://pkj-m.github.io/blog/feed.gsoc.xml"
+    project: 'radis'
+  siddharthlal25:
+    rss_feed: "https://siddharthlal25.github.io/feed.xml" # TODO (add blog with gsoc tag)
```

Comment:
@siddharthlal25 - could you make a feed that's only for gsoc? like what @pkj-m did above.